### PR TITLE
Release notes for 2.18.3

### PR DESCRIPTION
**Pull request by pleek91**
[Preview](https://github.com/PrefectHQ/prefect/blob/release-notes-2.18.3/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Prefect Release Notes
 
+## Release 2.18.3
+
+### Uncategorized
+- Wire up new engine to deployment runs — https://github.com/PrefectHQ/prefect/pull/12914
```
Comment (serinamarie) [inline RELEASE-NOTES.md:6]:
```suggestion
### Experimental
#### Engine
- Wire up new engine to deployment runs — https://github.com/PrefectHQ/prefect/pull/12914
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # Prefect Release Notes
 
+## Release 2.18.3
+
+### Experimental
+#### Engine
+- Wire up new engine to deployment runs — https://github.com/PrefectHQ/prefect/pull/12914
+
+### Fixes
+- Fix parameters that have unions types and a default values that become unresponsive and disappear from the ui — https://github.com/PrefectHQ/prefect-ui-library/pull/2355
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Fix parameters becoming unresponsive and disappearing in Prefect UI — https://github.com/PrefectHQ/prefect-ui-library/pull/2355
```
maybe?

Comment (serinamarie) [reply RELEASE-NOTES.md:10]: perhaps that's too catastrophic-sounding?

Comment (pleek91) [reply RELEASE-NOTES.md:10]: Yeah more concise 👍 I struggled to write that in a way that made sense. The details are in the PR description if anyone is interested.

Comment (serinamarie) [reply RELEASE-NOTES.md]: great!

**Review by serinamarie — APPROVED** [review]
lgtm!

**Pull request merged**